io-engine/src/subsys/nvmf/subsystem.rs

```diff
@@ -259,6 +288,99 @@ impl NvmfSubsystem {
         }
     }
 
+    /// Completion error callback for nexuses.
+    unsafe extern "C" fn nexus_cpl_error_cb(
+        req: *mut spdk_nvmf_request,
+        _cb_arg: *mut ::std::os::raw::c_void,
+    ) {
+        let req = &mut *req;
+        let cpl = req.nvme_cpl_mut();
+        let mut status = cpl.status();
+
+        if status.crd() == 0 {
+            return;
```

Comment:
This means we don't set crdt? So it's disabled?

Reply:
Either it was not configured by us, or Do Not Retry flag was set, or the host did not enable ACRE when target was configured. So we just stick with the flags SPDK sets in this case.